microsoft-edge/webview2/samples/webview2samplewincomp.md

```diff
@@ -73,159 +73,113 @@ Microsoft Visual Studio is required.  Microsoft Visual Studio Code is not suppor
 
    *  `<your-repos-directory>/WebView2Samples-main/SampleApps/WebView2SampleWinComp/WebView2SampleWinComp.sln`
 
-   A **Review Solution Actions** dialog box might appear:
+   A **Review Solution Actions** dialog box might open, prompting you for which installed Windows SDK to retarget the project to:
 
-   ![The 'Review Solution Actions' dialog box.](media/webview2samplewincomp-review-solution-actions.png)
+   ![The 'Review Solution Actions' dialog box.](webview2samplewincomp-images/review-solution-actions.png)
 
-1. Click the **OK** button.
+1. In the **Windows SDK Version** dropdown list, select **10.0.20348.0** or **10.0.18362.0** (but not 10.0.19041.0), and then click the **OK** button.  If those versions aren't available, do the steps in the "Install the Windows SDK" section below.  Otherwise, skip to the section below that.
 
+If the solution is already open, you can change the target as follows:
 
-<!-- ====================================================================== -->
-## Step 6 - Install workloads if prompted
-
-1. If prompted, install any Visual Studio workloads that are requested.  In a separate window or tab, see [Install Visual Studio workloads](../how-to/machine-setup.md#install-visual-studio-workloads) in _Set up your Dev environment for WebView2_.  Follow the steps in that section, and then return to this page and continue below.
-
-   Solution Explorer shows the **WebView2SampleWinComp** project.
-
-   <!-- Solution Explorer shows the **WebView2SampleWinComp** project: -->
-
-   <!-- ![The WebView2SampleWinComp sample opened in Visual Studio in Solution Explorer.](media/webview2samplewincomp-in-solution-explorer.png) -->
-   <!--todo: create png-->
+* In **Solution Explorer**, right-click the **WebView2SampleWinComp** project (not solution), and then click **Retarget Projects**.
 
 
 <!-- ====================================================================== -->
-## Step 7 - View the opened project
+## Step 6 - Install the Windows SDK
 
-The project opens in Visual Studio:
+By default, this sample app uses the latest Window 10 SDK version you have installed.  There's an issue with Windows 10 SDK, version 2004 (10.0.**19041**.0) that will stop this sample app from building.  If you run into this issue, either install (and retarget this project to use) a later version, such as Windows 10 SDK version 2104 (10.0.**20348**.0), or an earlier version, such as 10.0.**18362**.1.  
 
-![The WebView2SampleWinComp project in Visual Studio.](media/webview2samplewincomp-project-in-sln-explorer.png)
+To install a Windows 10 SDK:
```

Comment:
What about Windows 11?  Min worked with Win 11 and the 10.0.17763.0 Windows SDK version.

Reply:
If I understand, you used some version of Visual Studio to build this project targeting Win10 (not Win11), and then ran the resulting Win10 app on Win11.  17763 is a Win10 SDK:
https://developer.microsoft.com/en-us/windows/downloads/sdk-archive/

Reply:
Not sure if updating this article to target win11 is out of scope for this PR.